creator-node/sequelize/migrations/20200623002237-add-parent-file-data.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
```

Comment:
can you make sure to run this migration against a subset of prod data locally and confirm that it doesn't corrupt state? or at least make sure there are files of every "type" locally when you run this migration.

Reply:
Run on staging data locally w/ files of every "type"